api/tests/unit/application/preHandlers/assessment-supervisor-authorization_test.js

```diff
@@ -77,4 +77,61 @@ describe('Unit | Pre-handler | Assessment Supervisor Authorization', function ()
       });
     });
   });
+  describe('#verifyBySessionId', function () {
+    const request = {
+      headers: { authorization: 'VALID_TOKEN' },
+      params: {
+        id: 8,
+      },
+    };
+
+    beforeEach(function () {
+      sinon.stub(tokenService, 'extractTokenFromAuthChain');
+      sinon.stub(tokenService, 'extractUserId');
+      sinon.stub(supervisorAccessRepository, 'isUserSupervisorForSession');
+    });
+
+    describe('When user is the supervisor of the assessment session', function () {
+      it('should return true', async function () {
+        // given
+        const extractedUserId = 'userId';
+        tokenService.extractUserId.returns(extractedUserId);
+        tokenService.extractTokenFromAuthChain.returns('VALID_TOKEN');
+        supervisorAccessRepository.isUserSupervisorForSession.resolves(true);
+
+        // when
+        const response = await AssessmentSupervisorAuthorization.verifyBySessionId(request, hFake);
+
+        // then
+        sinon.assert.calledWith(supervisorAccessRepository.isUserSupervisorForSession, {
+          sessionId: request.params.id,
+          userId: extractedUserId,
+        });
+        expect(response).to.be.true;
+      });
+    });
+
+    describe('When user is not the supervisor of the assessment session', function () {
+      it('should return status code 401 and the error message', async function () {
```

Comment:
**nitpick**
`it('should return status code 401 and THE error message', async function () {`